draft-ietf-opsawg-pcapng.md

```diff
@@ -1478,7 +1478,7 @@ thread.
 The Enhanced Packet Block Flags Word is a 32-bit value that
 contains link-layer information about the packet.
 
-The word is encoded as an unsigned 32-bit integer, using the
+The word is encoded as a 32-bit unsigned integer, using the
```

Comment:
More bit-width redundancy? "...is a 32-bit value ... encoded as a 32-bit unsigned integer".